### ISSUE TITLE
Show cumulative directory size in status bar

### ISSUE THREAD

**Issue by hfel** [ranger/ranger#1887]
Hello! I would like to know if it's possible to show the cumulative size (files plus folders) of the current directory in the status bar as opposed to just the sum file size.

**Comment by toonn** [comment]
Maybe with the `autoupdate_cumulative_size` option or the command.

**Comment by hfel** [comment]
I set in the config and it made no difference. I believe this was discussed before in #540.

**Comment by toonn** [comment]
You set the option *and* ran `dc`?

**Comment by hfel** [comment]
Well, `dc` just shows the cumulative size for the active selection; I would like the size indicator in the _status bar_ in the bottom to show the cumulative size for the current folder as opposed to just the total file size.

**Comment by toonn** [comment]
Yeah but does it not after you press `dc`?

**Comment by hfel** [comment]
No, it only applies to the selected folders; there's no change in the status bar. In terms of functionality, of course I can just go up one level and `dc` the folder I want, but I would like to see it by default for the current folder in the status bar. I guess a bottleneck can be the indexing, as running `dc` on a large folder can visibly take some time.

**Comment by abitrolly** [comment]
`dc` updates left side of the status bar with the size of the current folder.

Left side of status bar before `dc`.
![image](https://user-images.githubusercontent.com/8781107/106362699-43fcf580-6335-11eb-876c-952aac7a6910.png)
Left side of status bar after `dc`.
![image](https://user-images.githubusercontent.com/8781107/106362712-5840f280-6335-11eb-8432-30c2b4885403.png)

The problem now that there is no way to get back to file count.
